src/components/date-picker-month/date-picker-month.tsx

```diff
@@ -272,7 +272,7 @@ export class DatePickerMonth {
     if (day - 6 === startOfWeek) {
       return days;
     }
-    for (let i = Math.abs(lastDate.getDay() - startOfWeek); i >= 0; i--) {
```

Comment:
Can you add a comment in the PR description about why this Math.abs is not needed?

Reply:
Sure.